templates/terraform/environments/stage/main.tf

```diff
@@ -102,11 +102,14 @@ module "stage" {
 
   domain_name = local.domain_name
 
-  # Instead of NAT gateway, enable NAT instance at cost of $1 per month
+  # NAT configuration
+  ## Instead of NAT gateway, enable NAT instance at cost of $1 per month
   vpc_enable_nat_gateway = false
 
-  # When use NAT gateway, this setting will save some money as there a cost associated to each NAT gateway, but if the AZ with the gateway
-  # goes down, nothing in the private subnets will be able to reach the internet. Not recommended for production.
+  ## Use this setting to override default NAT instance types
+  vpc_nat_instance_types = ["t3.nano", "t3a.nano"]
```

Comment:
Let's just remove this and use the default. It won't be important for most people.